libs/core/src/lib/utils/services/content-density.service.ts

```diff
@@ -15,15 +16,16 @@ export const DEFAULT_CONTENT_DENSITY: ContentDensity = 'cozy';
  */
 @Injectable()
 export class ContentDensityService {
-    private _contentDensity: BehaviorSubject<ContentDensity> = new BehaviorSubject(DEFAULT_CONTENT_DENSITY);
+    /** Content Density BehaviourSubject */
+    readonly contentDensity = new BehaviorSubject(DEFAULT_CONTENT_DENSITY);
```

Comment:
Previously it was private and only getter was revealed, now both of them are revealed, is there a reason for that?

Reply:
If you look closer, it's exactly the same: previously it was a getter for behavior subject, which served a purpose to make it read-only. This small refactoring just makes the syntax lighter, nothing is added, nothing is lost

Reply:
Yea, i see, thought that previously getter was returning an `Observable`.